addon/components/ember-notify/message.js

```diff
@@ -2,6 +2,10 @@ import Ember from 'ember';
 import layout from '../../templates/components/ember-notify/message';
 import Notify from 'ember-notify';
 
+function checkSendCloseIntent() {
+  if (this.get('isDestroyed')) return;
```

Comment:
You can add this to the `closeIntent` function itself, and then you don't need `checkSendCloseIntent`.